setup.py

```diff
@@ -119,6 +119,15 @@ def run(self):
                 npm = shutil.which("npm")  # this is required on windows
                 if npm is None:
                     raise RuntimeError("NPM is not installed.")
+
+                # Required when using NPM >3
+                log.info(f"> Installing rollup, react, and react-dom")
+                subprocess.run(f"{npm} install rollup".split(), cwd=js_dir, check=True)
+                subprocess.run(f"{npm} install react".split(), cwd=js_dir, check=True)
+                subprocess.run(
+                    f"{npm} install react-dom".split(), cwd=js_dir, check=True
+                )
+
```

Comment:
Since adding this put react and react-dom into package.json, I'm, not sure if these lines are needed anymore.

At the very least, rollup wasn't being automatically installed. Should I move rollup into `package.json` `devDependencies`?

Reply:
`react` and `react-dom` should be added to [`src/js/package.json`](https://github.com/idom-team/django-idom/blob/main/src/js/package.json#L20-L22). It's not really clear to me how this was working without those being declared?

Reply:
I have no idea man, but it's there now!